test/library/standard/Memory/Initialization/MoveInitializeCopyError.good

```diff
@@ -0,0 +1,2 @@
+MoveInitializeCopyError.chpl:4: In function 'test1':
+MoveInitializeCopyError.chpl:8: error: actual is copied when passed to 'rhs' of 'moveInitialize'
```

Comment:
I think it would be nicer if the error said something like this:

> error: calling 'moveInitialize' with 'r2' actual would result in a copy

Reply:
You can call `toString` on the actual Symbol to get r2

Reply:
What about for cases when the actual is an unnamed value?

Reply:
We can just print `<temporary>` or something which toString probably tries to do

Reply:
I'll make sure. Thanks!